kubernetes/launch-job.sh

```diff
@@ -67,8 +67,10 @@ usage() {
   echo "    for the cyclic-adversary. The corresponding model file is expected"
   echo "    to be in either the TF weights directory or /shared/victims/."
   echo "  --victim-ckpt CHECKPOINT"
-  echo "    Only for and is required for iterated training. Path to the TF weights"
-  echo "    directory for the last victim in the initial curriculum."
+  echo "    Only for and is required for iterated training. Path to the TF"
+  echo "    weights directory for the last victim (i.e., the bot not"
+  echo "    being trained) in the initial iteration's curriculum (CURRICULUM",
+  echo "    or ALTERNATE_CURRICULUM if --alternate-iteration-first is set)."
```

Comment:
Does the name of this checkpoint need to match the name given in the curriculum file?

Reply:
just needs to be a directory containing the correct TF weights for the model, I don't think the names have to match (e.g., things getting renamed should be fine)